### ISSUE TITLE
Write a (small) documentation!

### ISSUE THREAD

**Issue by Naereen** [nedbat/zellij#2]
Hi!
Your slides (and talk) about this is awesome, and the library seems very interesting, but right now it's kind of hard to know how to use it.
Could you document it? At least a little bit?
[Sphinx](http://sphinx-doc.org/) or [pdoc](https://github.com/BurntSushi/pdoc) can help.
Thanks in advance!

**Comment by nedbat** [comment]
@Naereen You are right, I should document it.  It was a side project, with no vision that it would be used by other people, so it might be a long wait though....!

**Comment by TonyFlury** [comment]
Installation instructions would be great ;-) I couldn't work out to resolve the cairo dependency (Ubuntu 16.04)
